src/get-data.ts

```diff
@@ -40,15 +49,34 @@ const getData = async (): Promise<PRData> => {
   if (info.repository && info.sender) {
     prData.actor = info.sender.login;
   } else {
-    throw new Error('Issue with context');
+    throw new Error('Error: Issue with context');
   }
 
-  const coverageData = getInput('coverageData');
-  const commentData = await getCoverage(coverageData);
+  let coverageData;
+
+  try {
+    coverageData = getInput('coverageData');
+  } catch (error) {
+    throwError(error, 'Failed to grab file ./coverage/lcov.info.');
+
+    return;
+  }
 
-  prData.coverage.lines = commentData.lines;
-  prData.coverage.functions = commentData.functions;
-  prData.coverage.branches = commentData.branches;
+  let commentData;
+
+  try {
+    commentData = await getCoverage(coverageData);
+  } catch (error) {
+    throwError(error, 'Failed to parse coverage from ./coverage/lcov.info. See configuration for instructions on how to upload coverage.');
```

Comment:
You're already throwing this error from the `getCoverage` function so we don't need to do it again.